tests/integration/Core/Content/Product/SalesChannel/Detail/AvailableCombinationLoaderTest.php

```diff
@@ -54,7 +54,7 @@ protected function setUp(): void
     public function testCombinationsAreInResult(): void
     {
         $context = Context::createDefaultContext();
-        $salesChanelContext = Generator::createSalesChannelContext($context);
+        $salesChanelContext = Generator::generateSalesChannelContext(baseContext: $context);
```

Comment:
```suggestion
        $salesChanelContext = Generator::generateSalesChannelContext($context);
```

optional! but not needed to use named parameter here